open-apparel-registry/open-apparel-registry
# Polish embed settings and map

## Overview

- Clean up layout and styles of Settings > Embed (and Settings tabs)
- Link embed OAR badge to OAR info site
- Move facilities count into "Facilities" search tab label
- Make sidebar narrower
- Maintain horizontal sidebar map layout at narrower viewport widths
- Make map legend narrower to accommodate narrower maps

## Demo

![image](https://user-images.githubusercontent.com/128699/120518631-c529cf00-c39f-11eb-8739-7333d9304590.png)

![image](https://user-images.githubusercontent.com/128699/120518557-b2af9580-c39f-11eb-8894-e2956c76c9dc.png)


## Notes

We should also hide the "Share this search" button in embed mode, but I couldn't determine the best way to implement that. Opened #1374.


## Testing Instructions

* Make sure Settings > Embed looks reasonable (for all embed permission levels)
* Make sure map and sidebar look reasonable at various viewport widths

## Checklist
